src/ceph-disk/ceph_disk/main.py

```diff
@@ -3001,10 +3001,19 @@ def start_daemon(
                 ],
             )
         elif os.path.exists(os.path.join(path, 'systemd')):
+            # ensure there is no duplicate ceph-osd@.service
+            command_check_call(
+                [
+                    'systemctl',
+                    'disable',
+                    'ceph-osd@{osd_id}'.format(osd_id=osd_id),
```

Comment:
any reason why trailing ".service" was omitted? This is the default, but we don't have 100% certainty that it will always be this way, and IMO it's better to specify the full form of the unit name instead of relying on systemd to fill in the blank.